docs/user-guide/global-configurations/image-promotion-policy.md

```diff
@@ -0,0 +1,136 @@
+
+6. Click **Save Changes**.
+
+---
+
+## Applying an Image Promotion Policy
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to apply an image promotion policy.
+{% endhint %}
+
+Here, you can decide the application(s) and environment(s) for which image promotion is allowed. 
+
+1. Go to the **Apply Policy** tab.
+
+2. Use the checkbox to select an application.
+
+3. Click the `Promotion Policy` dropdown within it and choose the policy you wish to apply.
+
+4. A confirmation dialog box would appear. Click **Confirm**.
+
+### Performing Bulk Action
+
+You can also apply a policy to more than one application. Simply use the checkboxes to select the applications. You can do this even if there are many applications spanning multiple pages. You will see a draggable floating widget.
+
+Moreover, there are three filters available to make it easier for you to make the selections easier:
+* Application
+* Environment
+* Policy
+
+---
+
+## Result
+
+### Selecting Image for Promotion
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have build & deploy permission or above (along with access to the application and target environment) to select an image for promotion.
+{% endhint %}
+
+Here, you can promote images to the target environment(s).  
+
+1. Go to the **Build & Deploy** tab of your application.
+
+2. Click the **Promote** button next to the pipeline eligible for image promotion.
+
+3. In the `Select Image` tab, you will see a list of images. However, you can use the **Show Images from** dropdown to decide the image you wish to promote. It can be an image either from the CI pipeline or from any one of the intermediate stages (environments).
+
+4. Use the **SELECT** button on the image, and click **Promote to...**
+
+5. Select one or more target environments using the checkbox.
+
+6. Click **Promote Image**. 
+
+The image will get promoted to the target environment and it will be visible in the list of images in that target environment. However, if the super-admin enforced an approval mechanism, an email notification will be sent to the approvers. 
+
+You may check the status of your request in the `Approval Pending` tab.
+
+### Approving an Image Promotion Request
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+User needs to be a direct promotion approver or a super-admin to approve an image promotion request.
+{% endhint %} 
+
+1. Go to the **Build & Deploy** tab of your application.
+
+2. Click the **Promote** button next to the pipeline that has direct promotion enabled.
```

Comment:
Click the Promote button next to the workflow. (If Image promotion is allowed for any environment in a workflow, a Promote button will appear next to the workflow.)